src/Symfony/Component/Yaml/Tests/InlineTest.php

```diff
@@ -246,6 +246,8 @@ public function getTestsForParse()
             array('{\'foo\': \'bar\', "bar": \'foo: bar\'}', array('foo' => 'bar', 'bar' => 'foo: bar')),
             array('{\'foo\'\'\': \'bar\', "bar\"": \'foo: bar\'}', array('foo\'' => 'bar', 'bar"' => 'foo: bar')),
             array('{\'foo: \': \'bar\', "bar: ": \'foo: bar\'}', array('foo: ' => 'bar', 'bar: ' => 'foo: bar')),
+            array('{foo:bar: "baz"}', array('foo:bar' => 'baz')),
```

Comment:
Actually, in master, it should still parse as before and generate a deprecation notice, and the behavior changed in 4.0.